server/src/lib/model/db/migrations/20200603213417-add-demographics-table.ts

```diff
@@ -0,0 +1,79 @@
+export const up = async function (db: any): Promise<any> {
+  return db.runSql(
+    `
+      CREATE TABLE ages (
+        id BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        age VARCHAR(255) NOT NULL
+      );
+
+      CREATE TABLE sexes (
+        id BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        sex VARCHAR(255) NOT NULL
+      );
+
+      CREATE TABLE demographics (
+        id BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        client_id CHAR(36) NOT NULL,
+        age_id BIGINT(20),
+        sex_id BIGINT(20),
+        updated_at DATETIME DEFAULT now(),
+        FOREIGN KEY (client_id) REFERENCES user_clients (client_id),
+        FOREIGN KEY (age_id) REFERENCES ages (id),
+        FOREIGN KEY (sex_id) REFERENCES sexes (id),
+        UNIQUE (client_id, age_id, sex_id)
+      );
+
+      CREATE TABLE clip_demographics (
+        clip_id BIGINT(20) UNSIGNED NOT NULL PRIMARY KEY,
+        demographic_id BIGINT(20) UNSIGNED NOT NULL,
+        FOREIGN KEY (clip_id) REFERENCES clips (id),
+        FOREIGN KEY (demographic_id) REFERENCES demographics (id)
+      );
```

Comment:
Will work on populating this tomorrow now that the proof of concept has passed initial review!

Reply:
Done!